lib/cart.rb

```diff
@@ -0,0 +1,59 @@
+require_relative "menu_item"
+require_relative "menu"
+require_relative "order_item"
+require_relative "customer"
+
+class Cart
+  attr_accessor :cart
+  attr_accessor :total_cost
```

Comment:
I always prefer one-liners so 
```
attr_accessor :cart, :total_cost
```
ATM, there are several examples of this in the code.